### PR TITLE
Bug Resolve : Giving suggestions for correct word also.

### PR DESCRIPTION
**Pull request by rajrajput79**
On click on confirm button:  Giving suggestions for correct word also.
It was done for testing purposes removing the code from the back and front end.

### CODE REVIEW

**Comment by rajrajput79** [comment]
Removing the code for correct word suggestion,

It was testing purposes only, to add the future color scheme in UI.

**Review by rajrajput79 — COMMENTED** [review]
It was done for testing purpose only

**Comment by rajrajput79** [comment]
Request closed by me, it was done for testing purpose

File: src/team3.html [hunk 1]
```diff
@@ -161,9 +161,6 @@
 				type: 'POST',
 				data: { 'word': selectedText, 'language':langaugetowrite },
 			})
-				.done(function (data) {
-					Output.textContent = data;
-				});
 		}
 		else
```
Comment (sujankandela) [inline src/team3.html:165]: Yes, Try and test for the local machine.

If possible can we give a red underline for incorrect words?

**Pull request merged**